src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,250 @@
+                        if (savedCommand[1].equals("1")) {
+                            tasks.get(listIndex).markAsDone();
+                        }
+                        System.out.println("\t" + tasks.get(listIndex++).toString());
+                        break;
+                    }
+                    case "D": {
+                        int separationIndexOfDeadline = userInput.indexOf('|');
+                        String byInput = userInput.substring(separationIndexOfDeadline + 1).trim();
+                        userInput = userInput.substring(4, separationIndexOfDeadline - 1);
+                        Task inputDescription = new Deadline(userInput, byInput);
+                        tasks.add(inputDescription);
+                        if (savedCommand[1].equals("1")) {
+                            tasks.get(listIndex).markAsDone();
+                        }
+                        System.out.println("\t" + tasks.get(listIndex++).toString());
+                        break;
+                    }
+                    }
+                }
+                printListIndex(listIndex);
+                drawLines();
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+            e.printStackTrace();
+        }
+        return listIndex;
+    }
+
+    public static void writeToFile() {
+        try {
+            FileWriter fw = new FileWriter("duke.txt");
+            for (Task task : tasks) {
+                fw.write(task.toFileString() + System.lineSeparator());
+            }
+            fw.close();
+        } catch (Exception e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+    }
+
+    public static void printListIndex(int listIndex) {
+        System.out.println("Now you have " + listIndex + ((listIndex > 1) ? " tasks" : " task") + " in the list");
+    }
+
+    private static void commandsAvailable() {
+        drawLines();
+        System.out.println("Commands available: list, help, done, todo, deadline, event, delete, find, save, bye");
+        System.out.println("The expected format of input values: ");
+        System.out.println("list - gives the list of data inputted");
+        System.out.println("help - this pulls out the commands available");
+        System.out.println("done x - x is the index of data that you want to mark as done");
+        System.out.println("todo x - x is the task description");
+        System.out.println("deadline x /by y - x is the task description and y is the deadline date");
+        System.out.println("event x /at y - x is the task description and y is the event date");
+        System.out.println("delete x - removes the task located at index x of the list");
+        System.out.println("find x - looks for task description with x included");
+        System.out.println("save - this saves the current list");
+        System.out.println("bye - this terminates the program");
```

Comment:
You can consider concatenating the Strings instead of calling `System.out.println` multiple times to improve readability.